superduperdb/base/superduper.py

```diff
@@ -24,26 +24,16 @@ def superduper(item: t.Optional[t.Any] = None, **kwargs) -> t.Any:
 
 
 def _auto_identify_connection_string(item: str, **kwargs) -> t.Any:
-    from superduperdb.base.build import build_datalayer
-
-    if item.startswith('mongomock://'):
-        kwargs['data_backend'] = item
+    if re_match(r'^[a-zA-Z0-9]+://', item) is None:
```

Comment:
please use `re.match`

Reply:
@jieguangzhou it is `re.match` method I have created alias which is being used since we are only using single method from the `re` package.